ml-agents/mlagents/trainers/tests/test_environments/test_simple.py

```diff
@@ -144,7 +144,7 @@ def test_simple():
                 extrinsic:
                     strength: 1.0
                     gamma: 0.99
-            vis_encode_type: default
+            vis_encode_type: simple
```

Comment:
If the defaults are working correctly, we should be able to remove these lines for configs where they're not relevant (like this one). If I do that, I still get an exception in check_param_keys(), so I think you need to remove `vis_encode_type` from the list there too

Reply:
Solved!